.vscode/settings.json

```diff
@@ -10,5 +10,6 @@
       "language": "typescriptreact",
       "autoFix": true
     }
-  ]
+  ],
+  "eslint.options.ignorePath": ".gitignore"
```

Comment:
This does not seem to be available as an ESLint configuration option. So it needs to be specified with every usage of ESLint. One such usage is here, for VSCode users.